src/main/java/core/basesyntax/dao/FileDao.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.dao;
+
+import java.util.List;
+
+public interface FileDao {
+    List<String> readFromFile(String fileName);
+
+    void writeToFile(String fileName, String reportString);

```

Comment:
```suggestion
    void writeToFile(String fileName, String content);
```
Do not forget Abstraction